src/pages/data-reporting/changelog/index.mdx

```diff
@@ -32,6 +32,10 @@ The developer preview version provides a preview of the upcoming changes to the
 
 - There's a new method, [**Tags Chat Usage**](/data-reporting/reports-api/v3.5/#chat-usage), which shows how many chats are marked with each listed tag.
 
+### Chats
+
+- The `surveys` method has been renamed to [**Forms**](/data-reporting/reports-api/v3.5/#forms), which shows the number of submitted chat forms along with the count of specific answers.
```

Comment:
I suggest writing this info as two sentences to make it easier to read :)
`The `surveys` method has been renamed to [**Forms**](/data-reporting/reports-api/v3.5/#forms). It shows the number of submitted chat forms along with the count of specific answers.`

Reply:
Generally I agree... but I took an example from documentation about renaming method in v3.4 :) I will improve it

Reply:
Both are correct, but let's make it easier read ;)